### PR TITLE
DBZ-4205 Add db-wide changeStream privilege

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-4205

### CODE REVIEW

File: examples/mongodb/1.8/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mongo:3.6
+FROM mongo:4.2
```
Comment (gunnarmorling) [inline examples/mongodb/1.8/Dockerfile:1]: Any particular reason for not using 4.4?

Comment (jpechane) [reply examples/mongodb/1.8/Dockerfile:1]: Yes, as  menitoned during the call, 4.4 requires intra replica set authentication (using key). When I crack this nut for 5.0 I can create 4.4 and backport it here. Now I belive it is important to get working image out ASAP

Comment (gunnarmorling) [reply examples/mongodb/1.8/Dockerfile:1]: OK, got it.

Comment (jpechane) [reply examples/mongodb/1.8/Dockerfile:1]: @gunnarmorling Here is the switch directly to 5.0 - https://github.com/debezium/docker-images/pull/271

**Pull request merged**